transformato/tests/test_charmm.py

```diff
@@ -0,0 +1,168 @@
+import os
+import pytest
+import warnings
+warnings.filterwarnings("ignore", module="parmed")
+
+
+@pytest.mark.rbfe
+@pytest.mark.charmm
+@pytest.mark.skipif(
+    os.getenv("CI") == "true",
+    reason="Skipping tests that cannot pass in github actions",
+)
+def test_run_28_1h1q_rbfe_production_with_CHARMM():
+    # Generating output for a run of the CDK2 Ligand System
+    from transformato import (
+        IntermediateStateFactory,
+        ProposeMutationRoute,
+        SystemStructure,
+        load_config_yaml,
+    )
+
+    from transformato.mutate import perform_mutations
+    from transformato.utils import run_simulation
+
+    configuration = load_config_yaml(
+        config="transformato/tests/config/test-28_1h1q_rbfe.yaml",
+        input_dir="data/",
+        output_dir="/site/raid4/johannes/test",
+    )
+
+    s1 = SystemStructure(configuration, "structure1")
+    s2 = SystemStructure(configuration, "structure2")
+    s1_to_s2 = ProposeMutationRoute(s1, s2)
+
+    s1_to_s2.propose_common_core()
+    s1_to_s2.finish_common_core()
+
+    mutation_list = s1_to_s2.generate_mutations_to_common_core_for_mol1()
+    print(mutation_list.keys())
+    i = IntermediateStateFactory(
+        system=s1,
+        configuration=configuration,
+    )
+    perform_mutations(
+        configuration=configuration,
+        i=i,
+        mutation_list=mutation_list,
+        nr_of_mutation_steps_charge=1,
+        nr_of_mutation_steps_cc=1,
+    )
+    # run_simulation(i.output_files, engine="charmm")
+
```

Comment:
I don't really like this. 
I can never run these tests since they depend on data that is stored locally on your server with a executable that is also not available.
I see why this is done like this, but I would rather prefer that we find a workaround here so that we can run one of these on GH.

Reply:
I agree, we need to find a way to make the tests more flexible!
- The files which are needed should be all in `transformato/data/` so that shouldn't be the problem.
-  About the executable; we need a way to modify the `simulation_charmm.sh` file because I, for example, have different names depending on which cluster I am using and currently thats not straight forward to change (I need to modifie the `simulation_charmm.sh` file and then reinstall TF. 
- Yes it would be good if we would have one testcase running on GH, unfortunately I am not sure how to do that, I tried to do it for one `openMM` job but that also failed for some reason (I already opened a pull request where I would like to work on the test functonalities https://github.com/wiederm/transformato/pull/54)